.github/actions/setup-postgres/action.yml

```diff
@@ -0,0 +1,12 @@
+name: Setup Postgres
+description: Setup Postgres
+author: 'timo@animo.id'
```

Comment:
You can change this to your email @sairanjitAW, so people know who to reach out to with question regarding this action

Reply:
Oh okay thanks updated now